<?xml version="1.0" encoding="UTF-8" ?>
<launch>
	<arg name="port" default="/dev/ttyUSB0"/>
	<arg name="remoteAddr" default="4109910A"/>
	<arg name="input_topic" default="/camera/color/image_raw"/>
  <arg name="resize" default="0.15"/>
  <arg name="use_text" default="True"/>


	<node pkg="xbee_communication" type="xbee_image_sender.py" name="xbee_image_sender"  output="screen">
    <param name="port" value="$(arg port)" />
    <param name="remoteAddr" value="$(arg remoteAddr)" />
    <param name="input_topic" value="$(arg input_topic)" />
    <param name="resize" value="$(arg resize)" />
    <param name="use_text" value="$(arg use_text)" />
	</node>

</launch>
